schemas/mod-source-record-manager/jobExecutionDeletionPair.schema

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Hello @SatishkumarPurswani, please change the file extension to ".json" to be consistent with other schema files in the repo. Thank you!

Reply:
Hi @RuslanLavrov - PR updated as per suggestions, Thanks 🙂